spec/factories/preference_sets.rb

```diff
@@ -2,5 +2,6 @@
   factory :preference_set do
     user
     case_volunteer_columns { {} }
+    table_state { {} }
```

Comment:
Just a note for when we pair, I'll teach you a bit about traits so we can easily populate this.